ssl/ssl_lib.c

```diff
@@ -4883,3 +4885,29 @@ uint32_t SSL_get_max_early_data(const SSL *s)
 {
     return s->max_early_data;
 }
+
+__owur unsigned int ssl_get_max_send_fragment(const SSL *ssl)
+{
+    /* Return any active Max Fragment Len extension */
+	if (ssl->server 
```

Comment:
The other side of the coin : the MFL extension is a request to the server-side that it limits the length of the frame it sends ...